### ISSUE TITLE
Override type `input` not working properly when combining with FormlyIonicModule

### ISSUE THREAD

**Issue by tlarrain** [ngx-formly/ngx-formly#2665]
**Description**
I'm creating a new template for my inputs following the guide. If I use the name `input`, it doesn't override the template completely, leaving a weird combination between an ion-input and my custom input style

**Minimal Reproduction**
I couldn't make the reproduction on Ionic on the Stackblitz because it uses an older version of Ionic. I'm using Ionic 5. I put my custom component in this Stackblitz for your reference. It's really simple.

https://ngx-formly-ui-ionic-bzweaz.stackblitz.io

**Your Environment**

- Angular version: ^10.0.14
- Formly version: ^5.10.6
- Ionic version: ^5.5.1

**Additional context**
If I put `input` as type, this is the result. A mix between my custom input, and the default ion-item styles:
![image](https://user-images.githubusercontent.com/7906220/104990551-dd99de00-59ea-11eb-82f7-5d570b4ffa7d.png)

On the other hand, if I use `n-input`, the result is as desired:
![image](https://user-images.githubusercontent.com/7906220/104990661-0ae68c00-59eb-11eb-8217-14ed3e5a227c.png)

I'm using TailwindCSS to put styles, but I don't think that's an issue here.

**Comment by aitboudad** [comment]
When overriding an existing type we do a merge, so the remove the default ion-item styles you must pass the `wrappers` property:

```patch
FormlyModule.forRoot({
  types: [{
    name: "input",
    component: InputComponent,
+   wrappers: []
  }]
}),
```

another solution: Override the ionic wrapper

**Comment by tlarrain** [comment]
This is not working for me. I also tried creating a wrapper and attached it to that type only, and doesn't work either. Only by overriding the main wrapper works, but it breaks all my other inputs that aren't of type `input`.

**Comment by aitboudad** [comment]
I've pushed a fix in [v5.10.12](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.12) release when `wrappers` is explicitly specified we should not do a merge with the existing config.
